shared/test/test_redis_table.rb

```diff
@@ -185,27 +193,27 @@ def test_expiration
     assert_equal [row1, updated_row2], table.to_a
 
     # Jump to next expiration time - nothing should be deleted
-    @redis.time_travel margin_time
+    sleep margin_time
     assert_equal [row1, updated_row2], table.to_a
 
     # Jump to just before expiration a third time
-    @redis.time_travel expire_time - (2 * margin_time)
+    sleep expire_time - (2 * margin_time)
     assert_equal [row1, updated_row2], table.to_a
 
     # Reset expiration by deleting a row
     table.delete([1])
     assert_equal [updated_row2], table.to_a
 
     # Jump to expiration time - nothing should be deleted
-    @redis.time_travel margin_time
+    sleep margin_time
     assert_equal [updated_row2], table.to_a
 
     # Jump to just before expiration a final time
-    @redis.time_travel expire_time - (2 * margin_time)
+    sleep expire_time - (2 * margin_time)
     assert_equal [updated_row2], table.to_a
 
     # Jump to expiration time - this time, everything should be gone
-    @redis.time_travel margin_time
+    sleep margin_time
     assert_equal [], table.to_a
   end
```

Comment:
I made a quick attempt to get this working, but it's not playing nice with `fakeredis` and a quick search hasn't turned up any documentation on this either.

I'm sure I can solve it given enough time, but may I propose that it shouldn't block this PR?